mkdocs.yml

```diff
@@ -1,24 +1,45 @@
 ---
+# Replace the fields below with relevant information.
+# See https://www.mkdocs.org/user-guide/configuration/
+# to learn more about the configuration options.
 site_name: Ansible project template
 site_url: https://ansible.readthedocs.io/PROJECTNAME
-repo_url: https://github.com/ansible/PROJECTNAME
+repo_url: https://github.com/ansible-community/project-template
 edit_uri: blob/main/docs/
 docs_dir: docs
 strict: true
 
 theme:
-  name: "ansible"
+  name: 'ansible'
 
 nav:
-  - index.md
-  - getting_started_user.md
-  - installing.md
-  - User Guide:
-    - user_guide/how_to_xxx.md
+  - Home: 'index.md'
+  - 'community_guide.md'
+    - 'community_guide.md#code-of-conduct'
+    - 'community_guide.md#communication'
+      - 'community_guide.md#asynchronous-channels'
+      - 'community_guide.md#real-time-channels'
+  - 'user_guide.md'
```

Comment:
I tried generating the docs locally from this PR and they won't build at all. @Andersson007 can you try a `mkdocs build` locally and see what I'm talking about?

I'm also unsure that we want every subsection heading showing up here (all the extra # extensions). But once you get the docs build working, I can take another look.